apps/examples/src/examples/custom-styles/FilterStyleUi.tsx

```diff
@@ -33,3 +48,30 @@ export const FilterStyleUi = track(function FilterStyleUi() {
 		</div>
 	)
 })
+
+/* 
+Introduction:
+This is a very simple example of how to create a custom ui for your custom style. We want
```

Comment:
Beware of using the word "simple" in docs. It's common that a user might be having a hard time with an example and docs that are overly enthusiastic ("it's easy! simple! my dog could do this!") can be aggravating.

```suggestion
This is a an example of how to create a custom ui for your custom style. We want
```